examples/contrastive/contrastive.cpp

```diff
@@ -28,6 +28,17 @@ int main(int argc, char ** argv) {
         params_amateur.prompt = argv[3];
     }
 
+    float alpha = 0.1;
+    float beta = 0.5;
+
+    if(argc >= 5){
```

Comment:
```suggestion
    if (argc >= 5) {
```
Same with the other 'if' statement.